ecl/eclagent/eclgraph.cpp

```diff
@@ -591,7 +591,8 @@ bool EclGraphElement::prepare(IAgentContext & agent, const byte * parentExtract,
             break;
         case TAKif:
             {
-                Owned<IHThorArg> helper = createHelper(agent, NULL);
+                assertex(!subgraph->owner);
```

Comment:
I changed it because it is more correct, and to avoid another cut and paste problem in the future.